cmd/nerdctl/build.go

```diff
@@ -180,3 +188,7 @@ func generateBuildctlArgs(clicontext *cli.Context) (string, []string, error) {
 
 	return buildctlBinary, buildctlArgs, nil
 }
+
+func isLocalBuild(clicontext *cli.Context) bool {
+	return len(strutil.DedupeStrSlice(clicontext.StringSlice("tag"))) > 0
```

Comment:
This logic doesn't seem correct, why not parse --output

Reply:
That's how I had it but figured it was easier to just check if a tag is specified for the output.  I'll update it.  Thanks!